docs/index.rst

```diff
@@ -0,0 +1,58 @@
+PyPCAP
+======
+This is a simplified object-oriented Python wrapper for libpcap -
+the current tcpdump.org version, and the WinPcap port for Windows.
+
+Example use:
+::
+
+    >>> import pcap
+    >>> for ts, pkt in pcap.pcap():
+    ...     print ts, `pkt`
+    ...
+
+Install
+--------
+
+This package requires:
+
+* libpcap-dev
+
+* python-dev
```

Comment:
If we start uploading also wheels we could also add a comment that these are not needed for the distributions where we support wheels.